### PR TITLE
[W5.8b][W13-A1]Dominic Kenn Lim

### PR DESCRIPTION
**Pull request by dominickenn**
Add more NFR to DeveloperGuide.

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. System should respond within 2 seconds.
+6. System should be be easy to understand for novice users.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:98]: Define "easy"

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -94,6 +94,10 @@ Use case ends.
...
+7. System should allow for data recovery in case of program error.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:99]: This can be considered as a functional requirement as well. It is a bit blur.

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -94,6 +94,10 @@ Use case ends.
...
+8. All tests should be successful before pushing changes.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:100]: Not sure if we can consider this as NFR! It is the best practise for sure.

**Comment by okkhoy** [comment]
@dominickenn 
NFR (and requirements in general) should be as precise as possible.
Avoid adjectives like `easy`, `fast`, `intuitive` etc.,
Instead, use precise statements like `response time should be 1 second`, `detailed help which shows all options should be present`

Some comments added. Please ack & close the PR after reading comments.

**Comment by dominickenn** [comment]
Acknowledged.

**Pull request closed without merging**